doc/onboarding-extras.md

```diff
@@ -78,6 +78,33 @@ Please use these when possible / appropriate
   git checkout $(git show -s --pretty='%T' $(git show-ref -d $(git describe --abbrev=0) | tail -n1 | awk '{print $1}')) -- test; make -j4 test
   ```
 
+### LTS/Version labels
+
+We use labels to keep track of which branches a commit should land on:
+
+* `dont-land-on-v?.x`
+  * For changes that do not apply to a certain release line
+  * Also used when the work of backporting a change outweighs the benefits
```

Comment:
ping @MylesBorins @nodejs/lts  @gib and I are going through https://github.com/nodejs/LTS/issues/216 now and hitting this issue. I think `backport-requested-to` PRs should have been omitted from the gist/issues in #216 above (or we should add `dont-land-on` labels to them, because dont-land-on get omitted).